arangod/Aql/GraphNode.cpp

```diff
@@ -246,10 +246,8 @@ GraphNode::GraphNode(ExecutionPlan* plan, ExecutionNodeId id,
 
     // First determine whether all edge collections are smart and sharded
     // like a common collection:
-    if (ServerState::instance()->isRunningInCluster()) {
-      _isSmart = _graphObj->isSmart();
-      _isDisjoint = _graphObj->isDisjoint();
-    }
+    _isSmart = _graphObj->isSmart();
+    _isDisjoint = _graphObj->isDisjoint();
```

Comment:
I'm not sure why this change is going in now, rather than when single server smart graphs were first merged into devel. I hope we didn't miss any use cases back then?

Reply:
At least for the GraphNode -> KShortestPaths inherticance speaking: This has been required - otherwise the Graph has not been identified as beeing a Disjoint in KPaths.